13_message_delivered/index.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Space+Mono&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="style.css" />
+    <title>Secret Door</title>
+  </head>
+  <body>
+    <div class="container">
```

Comment:
When building a form, it should be done with `<form>`. One advantage is that when you press Enter, it will automatically submit it. So you don't need to create an event listener to the submit button.

Here's a short tutorial about [form submit](https://developer.mozilla.org/en-US/docs/Web/API/HTMLFormElement/submit_event)